plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -140,7 +139,7 @@ func ClusterRoles() []rbac.ClusterRole {
 
 				// additional admin powers
 				rbac.NewRule("create").Groups(authorizationGroup).Resources("localsubjectaccessreviews").RuleOrDie(),
-				rbac.NewRule(ReadWrite...).Groups(rbacGroup).Resources("roles", "rolebindings").RuleOrDie(),
+				rbac.NewRule(ReadWrite...).Groups(rbac.GroupName).Resources("roles", "rolebindings").RuleOrDie(),
```

Comment:
@deads2k I do this because i found we have include the package` rbac "k8s.io/kubernetes/pkg/apis/rbac"`. Why we should keep package consistent? a little confusing. If we want keep consistent, this rule is better than use `"rbac.authorization.k8s.io"` everywhere directly. Do i missing something? :)